From d5c1b97cd1eb0af1711af415bbd677b0510c718b Mon Sep 17 00:00:00 2001
From: Tim Parenti <tim@timtimeonline.com>
Date: Thu, 8 Oct 2020 17:52:53 -0400
Subject: [PATCH 47/47] Fiji observes DST from 2020-12-20 to 2021-01-17

* australasia (Fiji): Guess that the later-than-usual start date is a
one-time departure from the recent pattern.
* NEWS: Mention this.
---
 NEWS        | 13 +++++++++++++
 australasia | 17 ++++++++++++++++-
 2 files changed, 29 insertions(+), 1 deletion(-)

diff --git a/NEWS b/NEWS
index c77f76d..298ad4d 100644
--- a/NEWS
+++ b/NEWS
@@ -1,5 +1,18 @@
 News for the tz database
 
+Unreleased, experimental changes
+
+  Briefly:
+    Fiji starts DST later than usual, on 2020-12-20.
+
+  Changes to future timestamps
+    Fiji will start DST on 2020-12-20, instead of 2020-11-08 as
+    previously predicted.  DST will still end on 2021-01-17.
+    (Thanks to Raymond Kumar and Alan Mintz.)  Assume for now that
+    the later-than-usual start date is a one-time departure from the
+    recent pattern.
+
+
 Release 2020b - 2020-10-06 18:35:04 -0700
 
   Briefly:
diff --git a/australasia b/australasia
index a8886ff..aa7ab7f 100644
--- a/australasia
+++ b/australasia
@@ -381,6 +381,19 @@ Zone	Indian/Cocos	6:27:40	-	LMT	1900
 # From Michael Deckers (2019-08-06):
 # https://www.laws.gov.fj/LawsAsMade/downloadfile/848
 
+# From Raymond Kumar (2020-10-08):
+# [DST in Fiji] is from December 20th 2020, till 17th January 2021.
+# From Alan Mintz (2020-10-08):
+# https://www.laws.gov.fj/LawsAsMade/GetFile/1071
+# From Tim Parenti (2020-10-08):
+# https://www.fijivillage.com/news/Daylight-saving-from-Dec-20th-this-year-to-Jan-17th-2021-8rf4x5/
+# "Minister for Employment, Parveen Bala says they had never thought of
+# stopping daylight saving. He says it was just to decide on when it should
+# start and end.  Bala says it is a short period..."
+# Since the end date is still in line with our ongoing predictions, assume for
+# now that the later-than-usual start date is a one-time departure from the
+# recent second Sunday in November pattern.
+
 # Rule	NAME	FROM	TO	-	IN	ON	AT	SAVE	LETTER/S
 Rule	Fiji	1998	1999	-	Nov	Sun>=1	2:00	1:00	-
 Rule	Fiji	1999	2000	-	Feb	lastSun	3:00	0	-
@@ -392,7 +405,9 @@ Rule	Fiji	2012	2013	-	Jan	Sun>=18	3:00	0	-
 Rule	Fiji	2014	only	-	Jan	Sun>=18	2:00	0	-
 Rule	Fiji	2014	2018	-	Nov	Sun>=1	2:00	1:00	-
 Rule	Fiji	2015	max	-	Jan	Sun>=12	3:00	0	-
-Rule	Fiji	2019	max	-	Nov	Sun>=8	2:00	1:00	-
+Rule	Fiji	2019	only	-	Nov	Sun>=8	2:00	1:00	-
+Rule	Fiji	2020	only	-	Dec	20	2:00	1:00	-
+Rule	Fiji	2021	max	-	Nov	Sun>=8	2:00	1:00	-
 # Zone	NAME		STDOFF	RULES	FORMAT	[UNTIL]
 Zone	Pacific/Fiji	11:55:44 -	LMT	1915 Oct 26 # Suva
 			12:00	Fiji	+12/+13
-- 
1.8.3.1

